browser/importer/brave_external_process_importer_host.cc

```diff
@@ -86,19 +89,24 @@ void BraveExternalProcessImporterHost::OnImportLockDialogEnd(bool is_continue) {
   }
 }
 
-bool BraveExternalProcessImporterHost::CheckForChromeLock(
-    const importer::SourceProfile& source_profile) {
-  if (source_profile.importer_type != importer::TYPE_CHROME)
+bool BraveExternalProcessImporterHost::CheckForChromeOrBraveLock() {
```

Comment:
Can we change the name to more general naming like CheckForBrowserLock?

Reply:
That's what I was going for, but I don't think it should have the fully generic `CheckForBrowserLock` name unless it also subsumes `CheckForFirefoxLock`. I could do do that (e.g. by making `FirefoxProfileLock` a subclass of `BrowserProfileLock`, etc.) but that would require patching and since it isn't necessary I left it out.